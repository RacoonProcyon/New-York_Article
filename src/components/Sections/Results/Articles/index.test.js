import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import ArticleListItem from './index';
import configureStore from 'redux-mock-store';
import thunk from 'redux-thunk'

const initialState = {
    searchData: [],
    pageNumber: 0,
    recentQuery: "",
    singleArticle: []
}
let middlewares = [thunk]
const mockStore = configureStore(middlewares)
let store = mockStore({ searchReducer: initialState })

it('renders correctly when there are no items', () => {
    const tree = renderer.create(<Provider store={store}>
        <ArticleListItem />
    </Provider>
    ).toJSON();
    expect(tree).toMatchSnapshot();
});