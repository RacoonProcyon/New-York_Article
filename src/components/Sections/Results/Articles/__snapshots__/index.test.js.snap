// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<article
  className="col-12 col-md-8 mt-3"
  id="articles"
>
  <p
    className="m-0"
  >
    Results:
  </p>
  <ul
    className="list-group"
  />
  <a
    href="/"
    onClick={[Function]}
  >
    Prev
  </a>
  <a
    className="float-right"
    href="/"
    onClick={[Function]}
  >
    Next
  </a>
</article>
`;
